internal/commands/user/delete/flags.go

```diff
@@ -0,0 +1,7 @@
+package delete
+
+const (
```

Comment:
with the soon-to-be-rebase, i've been merging this file with any `inputs.go` file we have